frontend/src/metabase/entities/actions/actions.ts

```diff
@@ -65,6 +69,7 @@ const enableImplicitActionsForModel =
           type: "implicit",
           kind: "row/create",
           model_id: modelId,
+          database_id: databaseId,
```

Comment:
I'm not entirely sure whet you do with this on the backend, but if you're persisting this on the action, if the database of the model is changed it make this out-of-sync.

Reply:
I mean it's a strange thing to do, but we let you do it.

![change model db](https://user-images.githubusercontent.com/30528226/220179122-c13c2bed-5482-4095-9cf7-4aff14b06b55.gif)

Reply:
Uh, I had no idea. Persisting the db_id is a terrible idea then...

Reply:
I've now updated it so it's not persisted on the BE. But I'm assuming the actions are cached as entities on the frontend, so this solution will mean the database ID of an action can get out of sync on the FE too, right?

Reply:
And if I only add the `database_id` to `ordered_cards` and nowhere else, would that solve the cache issue?

Reply:
I only added `database_id` to the `ordered_cards` like you originally suggested :) The actions in `ordered_cards` aren't cached, so the database_id doesn't get out of sync in the FE cache anymore